editor/src/core/property-controls/property-controls-parser.spec.ts

```diff
@@ -83,7 +85,7 @@ function runBaseTestSuite<T>(
       control: 'ham sandwich',
     }
     expect(parseFn(value)).toEqual(
-      left(objectFieldParseError('control', descriptionParseError('Not a member of an enum.'))),
+      left(objectFieldParseError('control', descriptionParseError(parseErrorDescription))),
```

Comment:
Fixed the tests because we switched from `parseEnum` to `parseConstant` in `property-controls-parser`